### PR TITLE
x-pack/filebeat/input/{websocket=>streaming}: rename package

### PR DESCRIPTION
**Pull request by efd6**
## Proposed commit message

See title.

---
This has been tested by injecting an `agentbeat` compiled with this version into a running 8.15 elastic-package stack and then running the proofpoint_on_demand integration, an integration that depends on the websocket input.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

- For #40264

### CODE REVIEW

File: x-pack/filebeat/docs/inputs/input-streaming.asciidoc
Comment (andrewkroh) [inline x-pack/filebeat/docs/inputs/input-streaming.asciidoc:1]: Does the inclusion point of this file need changed to keep the input list sorted alphabetically?

**Review by andrewkroh — APPROVED** [review]
LGTM

File: x-pack/filebeat/docs/inputs/input-streaming.asciidoc
Comment (efd6) [reply x-pack/filebeat/docs/inputs/input-streaming.asciidoc:1]: Yes, it does. https://github.com/elastic/beats/pull/40421/files/9bb3ef38b366af2bc256079a70c6fd1e0b819c61#diff-47e3c4d3920761aeb5007e72d9362444345b5003d06f1182f33ac26b520a9af7L94-L97 I'll fix that.

File: filebeat/docs/filebeat-options.asciidoc [hunk 1]
```diff
@@ -156,4 +156,4 @@ include::inputs/input-udp.asciidoc[]
 
 include::inputs/input-unix.asciidoc[]
 
-include::../../x-pack/filebeat/docs/inputs/input-websocket.asciidoc[]
+include::../../x-pack/filebeat/docs/inputs/input-streaming.asciidoc[]
```
Comment (andrewkroh) [inline filebeat/docs/filebeat-options.asciidoc:159]: I think this line needs moved up to ~L150.

Comment (efd6) [reply filebeat/docs/filebeat-options.asciidoc:159]: Yes

Comment (efd6) [reply filebeat/docs/filebeat-options.asciidoc]: I miss mechanically generated documentation rendering.

**Review by belimawr — APPROVED** [review]

**Pull request merged**